modal.js

```diff
@@ -1,14 +1,14 @@
 var modal = document.querySelector("#modal");
```

Comment:
if I wrap it in an self-invoking function, it doesn't work...am I missing something?